src/main/java/edu/cornell/kfs/module/cam/document/service/impl/CuAssetServiceImpl.java

```diff
@@ -53,7 +62,18 @@ public Asset updateAssetInventory(String capitalAssetNumber, String conditionCod
         Timestamp currentTimestamp = dateTimeService.getCurrentTimestamp();
         asset.setLastInventoryDate(currentTimestamp);
         asset.setLastUpdatedTimestamp(currentTimestamp);
-        return businessObjectService.save(asset);
+
+        AssetExtension assetExtension = (AssetExtension) asset.getExtension();
+        if (ObjectUtils.isNull(assetExtension)) {
+            assetExtension = new AssetExtension();
+            asset.setExtension(assetExtension);
+        }
+        Person person = personService.getPersonByPrincipalName(netid);
+        String lastScannedBy = person.getFirstName() + " " + person.getLastName() + " (" + netid + ")";
+        assetExtension.setLastScannedNetid(lastScannedBy);
```

Comment:
The database field and object name imply the data is a netid,  the value being populated is the name following by netid in parenthesis.  I suggest renaming the new values, or just storing the netid.

Reply:
also, the database field only supports 32, we would need more space if we want to store the name and net id